.github/ISSUE_TEMPLATE/new_team.yml

```diff
@@ -45,16 +52,19 @@ body:
     attributes:
       value: |
         ## Charter and Responsibilities
```

Comment:
I am not sure an issue would be the best place to add the charter of new teams. 
Maybe as an intake but charters IMO should be in a more permanently visible location like and MD document in the governance repo. 
Would that be the expectation? Open an issue for discussion/approval then add the charter?

Reply:
If we adopt something like #147 then yes, replacing this issue forms with a yaml template to edit in a PR makes sense. I see this as a stopgap until we've figured out how to formally document the teams.